Naive greedy alg/scheduling/Swarm.py

```diff
@@ -66,6 +80,12 @@ def _draw_stats(self, num_epoch, particle_idx, stats, config):
         print("Mean cost: ", df_stat["cost"].mean())
         print("Tuning configs:\n", config)
 
+    def _draw_graph(self, num_epoch, stats):
+        visualizer = Visualizer()
```

Comment:
The visualizer should not be part of the Swarm. Return instead your list of stats from run_epochs and visualize the result from the main.